funfact/algorithm.py

```diff
@@ -102,22 +132,28 @@ def _loss(fac, target):
                 new_best.append(b)
             best_fac.factors = new_best
 
-            if stop_by == 'best':
+            if stop_by == 'first':
                 if np.sum(converged) > 1:
```

Comment:
I just realize that we can stop already if `np.sum(converged) > 0` in this case

Reply:
Good point. I'll change that.